Kuzzle.Tests/API/Controllers/RealtimeControllerTest.cs

```diff
@@ -8,168 +8,155 @@
 using KuzzleSdk.API;
 using Moq;
 
-namespace Kuzzle.Tests.API.Controllers
-{
-    public class RealtimeControllerTest
-    {
-        private readonly RealtimeController _realtimeController;
-        private readonly KuzzleApiMock _api;
-
-        public RealtimeControllerTest()
-        {
-            _api = new KuzzleApiMock();
-            _realtimeController = new RealtimeController(_api.MockedObject);
-        }
-
-        [Fact]
-        public async void CountAsyncTest()
-        {
-            string roomId = "A113";
-            Int32 count = 3;
-            _api.SetResult(new JObject { {
-                "result", new JObject { {
-                    "count", count } }
-                } }
-            );
+namespace Kuzzle.Tests.API.Controllers {
```

Comment:
Automatically reformatted file because it didn't follow the styling rules as defined in this library solution file :expressionless: